sw-2020.html

```diff
@@ -155,7 +155,11 @@ <h2>
           Deliverables
         </h2>
 
-        <p><i>Draft state</i> indicates the state of the deliverable at the time of the charter approval. <i>Expected completion</i> indicates when the deliverable is projected to become a Recommendation, or otherwise reach a stable state.</p>
+<p><i>Draft state</i> indicates the state of the deliverable at the time of
+the charter approval. The Working Group intends to publish the latest
+state of their work as Candidate Recommendation Drafts and does not
```

Comment:
```suggestion
state of their work as Candidate Recommendation and does not
```
(I'm not sure there is any need to be so specific when CR Snapshots are also going to be used - cc @plehegar )